src/processErrors.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable max-len */
+const CORRECT_CASES = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+const ERRORS = {
+  MESSAGE_NO_TEXT: 'Text to convert is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
```

Comment:
This part `Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>"` duplicates in 2 messages